packages/tree-select/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@automattic/tree-select",
+	"version": "0.1.0",
+	"description": "A selector library for Redux",
+	"keywords": [
+		"redux",
+		"selector"
+	],
+	"author": "Ben Lowery <ben.lowery@automattic.com>",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-only",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"scripts": {
+		"prepare": "npm run build",
```

Comment:
`prepare` runs during `lerna bootstrap`, which is a companion to `npm install`. Let's run the build in the `npm run build`-like step, not in the `npm install`-ish one.

`npm install`-ish code should only set up `node_modules` and the various symlinks therein. And leave the build to the `build-client`, `build-packages`, `build-css` family of scripts.

Does that make sense?

Reply:
`prepare` makes the package ready for consumption. Doing it this way makes our usage of the package internal to calypso match the usage of an external consumer. In that way, we vet that our build process and package setup actually work. We also don't require any special calypso-only webpack config to use these packages from within calypso.